internal/game/pot_test.go

```diff
@@ -51,11 +51,11 @@ func TestDistributePot_SidePots(t *testing.T) {
 	util.InitLogger(true)
 
 	// Scenario: 3 players go all-in with different stack sizes.
-	// ShortStack (2000) has the best hand.
+	// YOU (2000) has the best hand.
 	// MidStack (5000) has the second best hand.
 	// BigStack (10000) has the worst hand.
 	// No low hands qualify.
-	playerNames := []string{"ShortStack", "MidStack", "BigStack"}
+	playerNames := []string{"YOU", "MidStack", "BigStack"}
```

Comment:
The first player name should be "YOU" at this moment. The restriction should be removed by refactoring.